store/store.go

```diff
@@ -69,6 +69,16 @@ const (
 	UbuntuCoreWireProtocol = "1"
 )
 
+type RefreshFlags int
+
+const (
+	_ = iota
+
+	// RefreshManaged indicates to the store that the refresh is
+	// managed via snapd-control.
+	RefreshManaged
```

Comment:
in the past we started with flags like this, and then realised the error of our ways; these days we're doing flags as a struct of bools. Easier to work with and reason about -- especially easier to read

Reply:
if you mean it to be a bitfield, you want this to be
```golang
const (
    RefreshManaged = 1<<iota
)
```

Reply:
Thanks, it sounds like the struct of cools^Wbools is the cooler^Wbetter approach. I will fix this and push a new PR.